aws/modules/privatebastion/module.ftl

```diff
@@ -0,0 +1,91 @@
+[#ftl]
+
+[@addModule
+    name="privatebastion"
+    description="Bastion access via SSM only"
+    provider=AWS_PROVIDER
+    properties=[
+        {
+            "Names" : "tier",
+            "Type" : STRING_TYPE,
+            "Description" : "The tier to use to host the private bastion",
+            "Default" : "msg"
```

Comment:
Not sure the msg tier is the best default tier for this

Reply:
I didn't want to introduce another tier just for this. Open to alternatives from the existing set ...

Reply:
Any that are in the existing set will already have the possibility of something already being deployed to them - by their nature they are common enough to be pre-defined. Providing one over another is going to enable some projects to deploy without disruption, whilst others will still need to. 

If you define your own by default, you can at least make it clear that this will create a new tier and subsequent subnet unless an override is provided. Otherwise its murky, and depends on a number of solution-specific factors as to the best tier.

Reply:
Within the existing set I'd suggest `dir` as the closest tier,  messaging is more aligned with the application where dir is more admin/infra side